konig-data-app-common/src/main/java/io/konig/data/app/common/ExtentContainer.java

```diff
@@ -209,9 +209,9 @@ public boolean validateQueryParam(String key, String value, URI datatype ) throw
 				Long.parseLong(value);
 				return true;
 			}
-			throw new DataAppException(errorMsg);
```

Comment:
You should delete the `errorMsg` variable since you are no longer using it.  (There are two instances of this variable that are never used.)